web/apps/labelstudio/src/config/Sentry.ts

```diff
@@ -5,10 +5,10 @@ import { Integrations } from "@sentry/tracing";
 import { Route } from "react-router-dom";
 
 export const initSentry = (history: RouterHistory) => {
-  if (APP_SETTINGS.debug === false) {
+  if (APP_SETTINGS.debug === false && APP_SETTINGS.frontend_sentry_dsn) {
     setTags();
     Sentry.init({
-      dsn: "https://5f51920ff82a4675a495870244869c6b@o227124.ingest.sentry.io/5838868",
+      dsn: APP_SETTINGS.frontend_sentry_dsn,
```

Comment:
```suggestion
      dsn: APP_SETTINGS.sentry_dsn,
```
The value as set in the template is `sentry_dsn` not `frontend_sentry_dsn`.

Reply:
ok.
What's the difference between `sentry_dsn` and `frontend_sentry_dsn`?
it's frontend code, so I thought `frontend_sentry_dsn`is right.